docker-for-windows/release-notes.md

```diff
@@ -50,6 +50,7 @@ This page contains information about the new features, improvements, known issue
 - Fixed an issue where the diagnostics were sometimes not uploaded correctly from the **Support** dialog.
 - Fixed DNS entries for `*.docker.internal` and Kubernetes cluster reset after the VM IP changes.
 - Fixed a corrupted internal cache which was preventing Docker Desktop from starting. Fixes [docker/for-win#8748](https://github.com/docker/for-win/issues/8748).
+- Fixed an issue where `docker info` sometimes took very long time. Fixes [docker/for-win#10675](https://github.com/docker/for-win/issues/10675)
```

Comment:
@StefanScherer Would it be possible to specify how long it previously took? 'Very long time' sounds vague. If we don't have 
```suggestion
- Fixed an issue where `docker info` sometimes took longer to respond. Fixes [docker/for-win#10675](https://github.com/docker/for-win/issues/10675)
```

Reply:
Sure, thank you!